vault/resource_auth_backend.go

```diff
@@ -168,6 +168,25 @@ func authBackendRead(ctx context.Context, d *schema.ResourceData, meta interface
 	if err := d.Set(consts.FieldAccessor, mount.Accessor); err != nil {
 		return diag.FromErr(err)
 	}
+
+	log.Printf("[DEBUG] Reading auth tune from %q", path+"/tune")
+	rawTune, err := authMountTuneGet(ctx, client, "auth/"+path)
```

Comment:
Does this require a Vault policy change? If yes, we should make sure to update the changelog accordingly and specifically call out a behavior change. See https://github.com/hashicorp/terraform-provider-vault/blob/main/CHANGELOG.md#500-may-21-2025 as an example.